docker-compose.yml

```diff
@@ -0,0 +1,172 @@
+version: "3.9"
+
+services:
+  rollups-node:
+    build:
+      context: .
+      dockerfile: build/Dockerfile
+      tags:
+        - "cartesi/rollups-node:devel"
+    entrypoint: ["cartesi-rollups-node", "validator"]
+    ports:
+      - "4000:4000"
+    depends_on:
+      hardhat:
+        condition: service_healthy
+      deployer:
```

Comment:
Just out of curiosity, we're still not at the point where the following setting would be necessary, right?

```
      hardhat_set_interval:
        condition: service_completed_successfully
```
https://github.com/cartesi/rollups-examples/blob/80be07fb68ba7048223c72f3f9fe1b2042ec9ae7/docker-compose.yml#L46-L47

Reply:
Not really, as we're still missing the other node services. But to be completely honest, I can't remember exactly the issue `hardhat_set_interval` was meant to solve. Nevertheless, it is a good idea to leave the configuration as close as possible to what we already have, so I'll add the dependency 😉